Cake.Recipe/Content/codecov.cake

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////////////////
+// TASK DEFINITIONS
+///////////////////////////////////////////////////////////////////////////////
+
+BuildParameters.Tasks.UploadCodecovReportTask = Task("Upload-Codecov-Report")
+    .WithCriteria(() => FileExists(BuildParameters.Paths.Files.TestCoverageOutputFilePath))
+    .WithCriteria(() => BuildParameters.IsMainRepository)
+    .WithCriteria(() => BuildParameters.CanPublishToCodecov)
+    .Does(() => RequireTool(CodecovTool, () => {
+        var settings = new CodecovSettings {
+            Files = new[] { BuildParameters.Paths.Files.TestCoverageOutputFilePath.ToString() },
+            Token = BuildParameters.Codecov.RepoToken,
```

Comment:
@AdmiringWorm quick question, just wanted to confirm....

If this hasn't been set, it won't stop things from working, right?  The token is only required when it is a private repository?